src/PrestaShopBundle/Command/ThemeCommand.php

```diff
@@ -0,0 +1,175 @@
+use PrestaShop\PrestaShop\Core\Addon\Theme\ThemeRepository;
+use PrestaShop\PrestaShop\Core\Addon\Theme\ThemeExporter;
+use Context;
+use Employee;
+use Shop;
+use Validate;
+
+/**
+ * Class ThemeCommand run command actions in CLI
+ */
+class ThemeCommand extends ContainerAwareCommand
+{
+    const ALLOWED_ACTIONS = array(
+        'install',
+        'enable',
+        'export',
+        'uninstall',
+    );
+
+    /**
+     * @var SymfonyStyle
+     */
+    protected $io;
+
+    /**
+     * @var int if the activation of the theme fails, return the right code
+     */
+    const RETURN_CODE_FAILED = 1;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('prestashop:theme')
+            ->setDescription('Manage your theme via command line')
+            ->addArgument('action', InputArgument::REQUIRED, sprintf('Action to execute (Allowed actions: %s).', implode(' / ', self::ALLOWED_ACTIONS)))
+            ->addArgument('theme', InputArgument::REQUIRED, 'Theme or path to zip on which the action will be executed')
+            ->addArgument('shop', InputArgument::OPTIONAL, 'Shop id on which the action will be executed');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function init(InputInterface $input, OutputInterface $output)
+    {
+        // the user must be allowed to enable themes
+        Context::getContext()->employee = new Employee(1);
+
+        $shopId = $input->getArgument('shop');
+        if ($shopId !== null) {
+            if (Validate::isLoadedObject($shop = new Shop($shopId))) {
+                Context::getContext()->shop = $shop;
+            } else {
+                $this->io->error(sprintf('The selected shop id "%s" is invalid', $shop));
+
+                return self::RETURN_CODE_FAILED;
+            }
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+        $action = $input->getArgument('action');
+        $theme = $input->getArgument('theme');
+        $this->init($input, $output);
+
+        if (!in_array($action, self::ALLOWED_ACTIONS)) {
+            $this->io->error(sprintf('Unknown theme action. It must be one of these values: "%s"', implode(' / ', self::ALLOWED_ACTIONS)));
+
+            return self::RETURN_CODE_FAILED;
+        }
+
+        switch ($action) {
+            case 'export':
+                $this->executeExportThemeAction($theme);
+                break;
+            case 'install':
+                if (!filter_var($theme, FILTER_VALIDATE_URL)
+                    && !file_exists($theme)) {
+                    $this->io->error('Zip file doesn\'t exist or url is not valid');
+
+                    return self::RETURN_CODE_FAILED;
+                }
+                $this->executeGenericThemeAction($action, $theme);
+                break;
```

Comment:
```suggestion
```

Reply:
Hello, why you suggest to remove these lines? we need to execute some action here...